hibernate-entitymanager/hibernate-entitymanager.gradle

```diff
@@ -30,3 +30,8 @@ sourcesJar {
 	manifest = null
 }
 
+javadocJar {
+	// The OSGi JAR manifest support does not like a non-existent classes dir,
+	// so make sure we dont use the OSGi one :)
+	manifest = null
+}
```

Comment:
Why publishing empty JavaDocs?

Reply:
I just replicated what was being done for the `sources` JAR. I could supress the generation of the JavaDoc JAR -- it's a simple change. I'll amend the PR.

Reply:
It is more a question.  No one on the team actually ever uses javadoc jars - in our opinion its silly when you can instead just use the sources jar.  So I am really asking.